VRDR/IJEMortality.cs

```diff
@@ -2971,6 +2972,7 @@ public string AUXNO2
             {
                 if (!String.IsNullOrWhiteSpace(value))
                 {
+					value = value.PadLeft(12 , '0');
```

Comment:
Please fix the spacing, set your editor to convert tabs to spaces so it's consistent.